### PR TITLE
Network connection

### PR DESCRIPTION
**Pull request by PippaClarkGDS**
## Why

Explains how a user creates a network connection to the DCS.

## What

Includes new steps and content to show how a user creates a network connection to the DCS.

## How to review

Check all technical details are correct. NB this has already been reviewed by 2 tech writers - if substantial changes are made, they will need to review again.

### CODE REVIEW

File: source/create-a-network-connection.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+Configure your chosen language library to use:
+
+* the DCS CA bundle
+* your client certificate
+* your private key
+
+Using the code you’ve written using your chosen library, make an HTTPS GET request to:
+
+```<DCS URL>/checks/passport```
```
Comment (PippaClarkGDS) [inline source/create-a-network-connection.html.md.erb]: @benjamin-mortimer checking with Information Assurance whether we can share this URL yet...

File: source/create-a-network-connection.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+The DCS and your service use mTLS to prove to each other that they possess the private key which corresponds to the public certificate. This extra layer of authentication means the DCS can be confident it only sends information to authorised clients.
+## Prerequisites
+Before setting up mTLS, you’ll need:
+
+* to [generate a private key and request a GDS-signed certificate](/generating-keys-and-csrs) before you can establish an mTLS connection - it can take up to 2 working days for the DCS to issue a certificate
+* the DCS Certificate Authority (CA) bundle - [request it from the DCS pilot helpdesk](/support) if you have not received it already
+
+## Test your connectivity to the DCS by making a GET request
+
+You need to have network access to the DCS. You can check you have this access by using a tool like [curl](https://curl.haxx.se/) or [Postman](https://www.postman.com/) to make a GET request to the DCS URL.
```
Comment (PippaClarkGDS) [inline source/create-a-network-connection.html.md.erb]: @benjamin-mortimer checking with Information Assurance whether we can share this URL yet...

File: source/create-a-network-connection.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+[Contact the DCS helpdesk](/support) if you do not have the DCS URL.
+
+There are many ways to check you have the correct access. For example, using the curl command line utility tool:
+
+```
+curl --cacert <path to CA bundle> \
+     --cert <path to client certificate> \
+     --key <path to private key> \
+     -v <DCS URL>/checks/passport
```
Comment (PippaClarkGDS) [inline source/create-a-network-connection.html.md.erb]: @benjamin-mortimer checking with Information Assurance whether we can share this URL yet...

**Comment by benjamin-mortimer** [comment]
This looks good to me. I think we can publish it with the URL placeholder and come back to update it later, if that's the only blocker.

**Comment by bjgill** [comment]
(this should be mergeable once Alex has removed the need for signed commits)

File: source/create-a-network-connection.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+< Date: Mon, 24 Feb 2020 15:51:42 GMT
+< Content-Type: application/json
+< Content-Length: 40
+< Connection: close
+< Vary: Accept-Encoding
+< Strict-Transport-Security: max-age=31536000
+<
+* Closing connection 0
+* TLSv1.2 (OUT), TLS alert, Client hello (1):
+{"available":true,"scheduledOutages":[]}%
```
Comment (Krenair) [inline source/create-a-network-connection.html.md.erb]: runaway percentage sign at the end?

**Comment by PippaClarkGDS** [comment]
Cheers Benjamin - will wait for Alex to remove the need for signed commits and then merge. 


All the best,

Pippa Clark
Technical Writer 
Government Digital Service

File: source/create-a-network-connection.html.md.erb (hunk 4 above)
Comment (bjgill) [reply source/create-a-network-connection.html.md.erb]: That's intentional - it indicates that the body does not have a newline at the end.

Comment (bjgill) [reply source/create-a-network-connection.html.md.erb]: Oh. Hang on. This is actually zsh being clever. It won't appear for people not using zsh, so should probably be removed.

```suggestion
{"available":true,"scheduledOutages":[]}
```

(https://stackoverflow.com/questions/29497038/why-does-a-curl-request-return-a-percent-sign-with-every-request-in-zsh)

Comment (PippaClarkGDS) [reply source/create-a-network-connection.html.md.erb]: have amended this in a commit

**Pull request merged**